common/app/views/fragments/items/elements/facia_cards/title.scala.html

```diff
@@ -4,29 +4,84 @@
 @import views.support._
 @import layout.FrontendLatestSnap
 @import implicits.ItemKickerImplicits._
+@import experiments.{ActiveExperiments, RemoveKickerSlashes}
 
 @icon = {
-    @if(header.isExternal) { @fragments.inlineSvg("external-link", "icon") }
-    @if(header.quoted) { @fragments.inlineSvg("garnett-quote", "icon") }
+@if(header.isExternal) {
```

Comment:
formatting change added automatically by IntelliJ; I'm assuming that these formatting settings are shared so makes sense to leave the change in to allow using the formatter in the future without diffs.